Tests/UnitTests/UnitTests.xcodeproj/project.pbxproj

```diff
@@ -655,7 +655,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_MODULES = NO;
 				CLANG_ENABLE_OBJC_ARC = YES;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
The first one is a subcategory for the second one. No need to specify value for a subcategory if the parent is set to that value.